Created Constants & Bar Graph Functions

johnsbuck opened the pull request:
Constants
=========
Two constants were added to the project for simplicity and later use,
the Traffic Monitor Channel (TMC) codes and the US Accidents data
headers.

  - The TMC codes corresponds an unsigned integer value to a English description that may be extrapolated later on.
  - The headers are just descriptions for the columns and created for ease of access.

Bar Graph Functions
===================
In the data, there is a Severity column that is used to indicate how it
has impacted traffic. The functions added to main.py create a bar graph
showing the population for each severity. One function simply creates
a bar graph and the other creates a logarithmic bar graph due to some
discrepancies in height.

Paul-Kyle-Turner commented on .idea/US_Accidents.iml:
```diff
@@ -0,0 +1,11 @@
+  <component name="TestRunnerService">
+    <option name="PROJECT_TEST_RUNNER" value="Unittests" />
+  </component>
+</module>
```
Is .idea needed? This is for pycharm and not really needed for public use.

johnsbuck replied:
.idea has been removed and added to .gitignore

johnsbuck replied:
What .idea? I don't know what you are talking about.

johnsbuck committed:
Created Constants & Bar Graph Functions

Constants
=========
Two constants were added to the project for simplicity and later use,
the Traffic Monitor Channel (TMC) codes and the US Accidents data
headers.

  - The TMC codes corresponds an unsigned integer value to a English description that may be extrapolated later on.
  - The headers are just descriptions for the columns and created for ease of access.

Bar Graph Functions
===================
In the data, there is a Severity column that is used to indicate how it
has impacted traffic. The functions added to main.py create a bar graph
showing the population for each severity. One function simply creates
a bar graph and the other creates a logarithmic bar graph due to some
discrepancies in height.
